### DISCUSSION TITLE
read access violation after contact callback

### DISCUSSION THREAD

**Discussion by AgentCodeBanks** [NVIDIA-Omniverse/PhysX#163] [Q&A]
I simulate a conveyor belt in PhysX by applying a velocity to each object that has a collision with the conveyor belt. If I now convexly decompose the transported objects (short U-profiles) with the V-HACD v4.0 (https://github.com/kmammou/v-hacd) with relatively high resolution, my program crashes because I get a read access violation exeption when calling the method "PxU32 pairFlags = si->getPairFlags()" in the ScScene.cpp file, line 5465. This only occurs when I have the GPU pipeline enabled. 

![grafik](https://github.com/NVIDIA-Omniverse/PhysX/assets/134261456/bb540b3d-fb4e-4e65-8d9a-837a32607748)

I suspect that the GPU is limited to a certain number of objects and the V-HACD creates too many small convex meshes. 

Does anyone know how I can prevent this without changing the resolution of the V-HACD?

**Comment by vreutskyy** [comment]
Hello
It's hard to tell from this description what's going on there. It would help if you could provide a repro sample (as a compilable snippet).
Thanks

**Comment by AgentCodeBanks** [comment]
Hello, 
I have attached the mesh as a .dat file and a snippet on how we load the file and add it to the PhysX.

[badMesh.zip](https://github.com/NVIDIA-Omniverse/PhysX/files/12143409/badMesh.zip)

**Reply by vreutskyy** [reply]
Hello. Thanks for your data, but to debug the crash I need a compilable sample that actually crashes when run, so I could run it in debug and see what's happening.
